src/ApiService/ApiService/onefuzzlib/Queue.cs

```diff
@@ -30,6 +34,12 @@ public class Queue : IQueue {
         _log = log;
     }
 
+    public async IAsyncEnumerable<QueueItem> ListQueues(StorageType storageType) {
+        var queueServiceClient = await GetQueueClientService(storageType);
+        await foreach (var q in queueServiceClient.GetQueuesAsync()) {
```

Comment:
this can return `queueServiceClient.GetQueuesAsync()` directly rather than yielding each item

Reply:
like this ?
![image](https://user-images.githubusercontent.com/7307755/191618368-59cd2039-2794-48f5-ad26-51ebbf7dcaaf.png)

Reply:
I think if you suggestion works - we can do this in ORM as well